src/components/store/slice.ts

```diff
@@ -0,0 +1,36 @@
+import { createSlice, type PayloadAction } from '@reduxjs/toolkit';
+import type { User } from '../../utils/types';
+
+const initialState: User = {
+  name: '',
+  age: 0,
+  eMail: '',
+  password: '',
+  gender: 'female',
+  photo: '',
+  country: '',
+};
+
+const formSlice = createSlice({
+  name: 'form',
+  initialState,
+  reducers: {
+    setFormData: (state, action: PayloadAction<User>) => {
```

Comment:
collect data from both forms at the same time and render on UI all data, not just from the last form